admin/spec/service/nca-tools-user.service.spec.js

```diff
@@ -115,51 +115,4 @@ describe('nca-tools-user.service', () => {
       expect(userDataService.sqlUpdateSchool).not.toHaveBeenCalled()
     })
   })
-
-  describe('recordLogonAttempt', () => {
```

Comment:
just so i understand the bigger picture here - is this recorded elsewhere, in adminLogonEvent for example?

Reply:
The recordLogonAttempt was the only piece of code using ncaToolsSession, and there is no code that calls it. I believe it is recorded in admin-logon-event (as in `authentication/local-strategy.js` @98-115 and `authentication/nca-tools-authentication-strategy.js` @52 and @60).

Reply:
this is what i assumed to be the case.